src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/Utilities/BaseTest.cs

```diff
@@ -75,6 +79,67 @@ public void BeforeAllTests ()
 				} catch (Exception ex) {
 					Console.Error.WriteLine ("Failed to determine whether there is Android target emulator or not: " + ex);
 				}
+
+				CreateBinutilsSymlink ();
```

Comment:
Why is this symlink creation now required? I'm assuming there were some test failures that popped up that required this addition?

Reply:
It's required to locate the in-tree binutils which only exist in our "installation" directory in the local build, they don't exist in e.g. `bin/TestRelease/blah`. The installation directory is relative to `XABT.dll` location, which will be in `bin/TestRelease/blah` but the `binutils` subdir won't be there - that's why we need the symlink